CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- Deactivated asset profile on delisting (Yahoo Finance)
```

Comment:
`Deactivated asset profiles automatically on delisting in the _Yahoo Finance_ service` ❓